docs/conf.py

```diff
@@ -6,51 +6,51 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath('..'))
-import webargs
+sys.path.insert(0, os.path.abspath(".."))
+import webargs  # flake8: noqa
```

Comment:
[fun story about `# flake8: noqa`](https://gitlab.com/pycqa/flake8/merge_requests/219)

I _believe_ you want `# noqa` here (or `# flake8: noqa` on a line by itself if you actually don't want to lint this file)